dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -150,7 +149,7 @@ public final Response getContainers(@Context final HttpServletRequest request,
     @Path("/{containerId}/uuid/{uuid}/content/{contentletId}")
     public final Response containerContent(@Context final HttpServletRequest req, @Context final HttpServletResponse res,
             @PathParam("containerId") final String containerId, @PathParam("contentletId") final String contentletId,
-            @PathParam("uuid") final String uuid) throws DotDataException, DotSecurityException, ParseErrorException,
+            @PathParam("uuid") final String uuid) throws DotDataException, ParseErrorException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.apache.velocity.exception.ParseErrorException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)